src/rgw/rgw_lc.cc

```diff
@@ -1554,11 +1555,11 @@ int RGWLC::bucket_lc_process(string& shard_id, LCWorker* worker,
     worker->workpool->drain();
   }
 
-  ret = handle_multipart_expiration(bucket.get(), prefix_map, worker, stop_at, once);
+  ret = handle_multipart_expiration(this, bucket.get(), prefix_map, worker, stop_at, once);
   return ret;
 }
 
-int RGWLC::bucket_lc_post(int index, int max_lock_sec,
+int RGWLC::bucket_lc_post(const DoutPrefixProvider *dpp, int index, int max_lock_sec,
```

Comment:
Same here, the dpp here is not needed, it should be a `this` since `RGWLC` is a `DoutPrefixProvider`.